tools/RAiDER/models/hrrr.py

```diff
@@ -228,12 +232,29 @@ def __init__(self):
         x0 = 0
         y0 = 0
         earth_radius = 6371229
-        p1 = CRS(f'+proj=lcc +lat_1={lat1} +lat_2={lat2} +lat_0={lat0} '\
+        self._proj = CRS(f'+proj=lcc +lat_1={lat1} +lat_2={lat2} +lat_0={lat0} '\
                  f'+lon_0={lon0} +x_0={x0} +y_0={y0} +a={earth_radius} '\
                  f'+b={earth_radius} +units=m +no_defs')
-        self._proj = p1
-
         self._valid_bounds = HRRR_CONUS_COVERAGE_POLYGON
+        self.setLevelType('ml')
+
+
+    def setLevelType(self, levelType):
```

Comment:
Isn't this defined in `weatherModel.py`? If not, we should move it over there so it can be available to other models.